app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -94,22 +96,38 @@ class AdministratorControlsActivityTest {
 
   @Test
   fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
```

Comment:
Please finx indentation in this entire file.

Reply:
resolved.